tools/SandcastleRefMapper/DocFx.SandcastleRefMapper.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net472</TargetFramework>
+    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+    <Product>SandcastleRefMapper</Product>
+    <Company>Microsoft</Company>
+    <Authors>Microsoft</Authors>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.DocAsCode.Metadata.ManagedReference" Version="2.40.12" />
```

Comment:
Change `Microsoft.DocAsCode.Metadata.ManagedReference` and `Microsoft.DocAsCode.Build.Engine` from `PackageReference` to `ProjectReference`.